docs/logs/perf_N_neighbors_physical_2020_12_31.log

```diff
@@ -0,0 +1,186 @@
+./build/tests/aca_tests --gtest_filter=*1_port_CREATE_plus_10_l2_neighbor_CREATE* -n [number of L2 neighbors to create]
+build = release
```

Comment:
Do we need to put this perf log file on github?

Reply:
it doesn't hurt for record keeping :)